src/shared/components/mutationTable/MutationTable.tsx

```diff
@@ -86,6 +86,7 @@ export interface IMutationTableProps {
     initialSortDirection?:SortDirection;
     paginationProps?:IPaginationControlsProps;
     showCountHeader?:boolean;
+    showAllExon?:boolean;
```

Comment:
same here, no need to add a prop. This ofc is also a good way to do it, but just because we've solved this using customizations in PatientViewMutationTable before it's better to follow the same pattern

Reply:
yes i see, if i don't add the new prop here, do you think it's ok to have `false` here by default?
```
render: (d:Mutation[]) => (this.props.genomeNexusCache
       ? ExonColumnFormatter.renderFunction(d, this.props.genomeNexusCache, false)
       : <span></span>),
```

Reply:
@leexgh yes fine to use false, or don't pass a third argument. Since it's an optional argument, it will be undefined and will work as expected